common/views/components/ImageGalleryV2/ImageGalleryV2.js

```diff
@@ -30,6 +31,12 @@ class ImageGallery extends Component<Props, State> {
   }
 
   showAllImages = () => {
+    ReactGA.event({
+      category: 'component',
+      action: `ImageGallery:click`,
```

Comment:
We don't use camel case - 1 way to do things?

Reply:
We've started using `PascalCase` in GA apparently – that's what I was being consistent with.

Reply:
Got it!
So what about action = `ReactComponentName:action`?
Action feels like it could be more granular. What happens if we have a hide all image?

Reply:
dunno – not sure we're going to solve it before we have the conversation though (but should probably start tracking something before then?)

Reply:
My concern is that `click` doesn't  say very much and might incorporate, or be mistaken for, other actions. We use other things besides click, so we can.

Reply:
Deemed good enough for now after conversation with @hthair

Reply:
> wouldn't leave much to the imagination?

Not sure how much imagination plays in collecting data, analysing it though... 🔮